site/app/views/grading/ElectronicGraderView.php

```diff
@@ -227,23 +248,67 @@ public function summaryPage($gradeable, $rows, $graders) {
                 <td>{$row->getUser()->getId()}</td>
                 <td>{$row->getUser()->getDisplayedFirstName()}</td>
                 <td>{$row->getUser()->getLastName()}</td>
-                <td>{$row->getGradedAutograderPoints()} / {$row->getTotalAutograderNonExtraCreditPoints()}</td>
+HTML;
+
+                if($show_auto_grading_points) {
+                    if ($highest_version != 0) {
+                        $return .= <<<HTML
+                <td>{$row->getGradedAutograderPoints()}&nbsp;/&nbsp;{$row->getTotalAutograderNonExtraCreditPoints()}</td>
+HTML;
+                    }
+                    else {
+                        $return .= <<<HTML
+                <td></td>
+HTML;
+                    }
+                }
+                if ($highest_version != 0) {
+                    $return .= <<<HTML
                 <td>
 HTML;
-                if ($row->beenTAgraded()) {
-                    $btn_class = "btn-default";
-                    $contents = "{$row->getGradedTAPoints()} / {$row->getTotalTANonExtraCreditPoints()}";
+                    $box_background = "";
+                    if ($row->getActiveDaysLate() > $row->getAllowedLateDays()) {
+                        $box_background = "background:#d9534f;padding:3px;margin-top:3px;margin-bottom:3px;margin-left:5%;margin-right:5%;border-radius:4px;";
```

Comment:
can you just make this a class that gets set on the column instead of defining this inline.